get-started-checkpoints/code-checkpoints/dvclive/src/train.py

```diff
@@ -0,0 +1,85 @@
+import tensorflow as tf
+import numpy as np
+import os
+
+from util import load_params, history_to_csv, history_list_to_csv
+import models
+from dvc.api import make_checkpoint
+import dvclive
+
+MODEL_DIR = "models/fashion-mnist/"
+MODEL_FILE = os.path.join(MODEL_DIR, "model.h5")
+DATA_DIR = "data/fashion-mnist"
+
+class DVCLiveCallback(tf.keras.callbacks.Callback):
+    def __init__(self, model_file = None):
+        super().__init__()
+        self.model_file = model_file
+
+    def on_epoch_end(self, epoch, logs=None):
+        logs = logs or {}
+        if self.model_file:
+            self.model.save(self.model_file)
+        for metric, value in logs.items():
+            dvclive.log(metric, value)
+        dvclive.next_step()
```

Comment:
Now that https://github.com/iterative/dvclive/pull/69 has been merged, we could replace this with:

```suggestion
from dvclive.keras import DvcLiveCallback

MODEL_DIR = "models/fashion-mnist/"
MODEL_FILE = os.path.join(MODEL_DIR, "model.h5")
DATA_DIR = "data/fashion-mnist"
```
Right?